package.json

```diff
@@ -133,6 +133,8 @@
     "rimraf": "^3.0.2",
     "source-map-loader": "^3.0.1",
     "storybook": "^8.0.4",
+    "style-loader": "^4.0.0",
+    "terser-webpack-plugin": "^5.3.10",
```

Comment:
How was this working before? Do these plugins actually come bundled with Webpack? Is it a good idea to install them separately?

Reply:
Per the docs it seems advisable to install separately so I added it https://webpack.js.org/plugins/terser-webpack-plugin/#root